internal/cmd/kafka/confluent_kafka_configs.go

```diff
@@ -71,6 +71,14 @@ func getConsumerConfigMap(group string, kafka *configv1.KafkaClusterConfig, clie
 		return nil, err
 	}
 	log.CliLogger.Debugf("Created consumer group: %s", group)
+	if commitGroupOffset {
+		if err := configMap.SetKey("enable.auto.commit", true); err != nil {
+			return nil, err
+		}
```

Comment:
The documentation says this is the default, pretty sure we can remove it. https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md
```suggestion
```